ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml

```diff
@@ -0,0 +1,45 @@
+canonical_go_repository: github.com/redhat-developer/odo
+
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+
+base_images:
+  cli:
+    name: '4.0'
+    namespace: ocp
+    tag: cli
+  golang:
```

Comment:
You don't need this -- these are base images for output image builds, you get golang as a build root as configured above

Reply:
ack thanks